src/zipMany.js

```diff
@@ -0,0 +1,37 @@
+var write = require('./write'),
+    geojson = require('./geojson'),
+    JSZip = require('jszip');
+
+module.exports = function(gjArray, options) {
+
+    var zip = new JSZip(),
+        layers = zip.folder(options && options.folder ? options.folder : 'layers');
+
+    gjArray.forEach(function(gj, layerIndex) {
+      [geojson.polygon(gj)]
+          .forEach(function(l) {
```

Comment:
* lines and points are now silently ignored. throw at least error saying that they are not supported if there any (`if (geojson.point(gj).geometries.length) throw new Error("point are not supported")`)
* fix indent: 2 -> 4